core/src/main/java/hivemall/fm/Feature.java

```diff
@@ -383,4 +383,10 @@ public static void l2normalize(@Nonnull final Feature[] features) {
         }
     }
 
+    @Override
```

Comment:
Why this `equals` method is required? Assume this is not used.

Reply:
I use it in a test in `CofactorizationUDTF` where I check if a `Feature` read from disk is equal to the expected `Feature`

Reply:
See https://medium.com/codelog/overriding-hashcode-method-effective-java-notes-723c1fedf51c 

Usually, overriding `equals` required `hashCode` because hashCode (and equals) is used for HashMap key search.